src/scripts/settings/Settings.ts

```diff
@@ -106,6 +106,14 @@ Settings.add(new MultipleChoiceSetting('backgroundImage', 'Background image:',
     ],
     'background-day'
 ));
+Settings.add(new MultipleChoiceSetting('eggAnimation', 'Egg Hatching Animation:',
+    [
+        new SettingOption('None', 'none'),
+        new SettingOption('Fully ready', 'full'),
+        new SettingOption('Start wiggling at 50 steps left', 'almost'),
```

Comment:
Could we make this a little bit shorter, maybe something similar to:
```suggestion
        new SettingOption('Almost & Fully ready', 'almost'),
```
```suggestion
        new SettingOption('Almost ready', 'almost'),
```

Reply:
Done! I went with `Almost & fully ready` (lowercase f for readability) just to make it clear that you get both animations instead of just when they're almost ready